src/acquisition/covidcast/signal_dash_data_generator.py

```diff
@@ -184,6 +184,13 @@ def get_latest_issue_from_metadata(dashboard_signal, metadata):
     max_issue = df_for_source["max_issue"].max()
     return pd.to_datetime(str(max_issue), format="%Y%m%d").date()
 
+def get_earliest_issue_from_metadata(dashboard_signal, metadata):
```

Comment:
Yes it been used in delphi-epidata/tests/acquisition/covidcast/test_signal_dash_data_generator.py to test if it can get the earliest issue from metadata.

Reply:
hmmm if the only use of get_earliest_issue_from_metadata is the test of get_earliest_issue_from_metadata, then we should probably drop it. unless you are planning to use get_earliest_issue_from_metadata in a future PR?

Reply:
Gotcha! I don't think it will be use in future...I will drop it now.